ops/model.py

```diff
@@ -3413,6 +3426,10 @@ def status_set(self, status: str, message: str = '', *, is_app: bool = False) ->
         """
         if not isinstance(is_app, bool):
             raise TypeError('is_app parameter must be boolean')
+        if not isinstance(message, str):
+            raise TypeError('message parameter must be a string')
```

Comment:
Is there any point having this, given the check immediately below?

Reply:
This checks the message argument, while the following check is for the status argument

Reply:
Oops, totally overlooked that :)